code/modules/mob/living/carbon/xenomorph/hive_datum.dm

```diff
@@ -202,6 +202,10 @@
 
 	HS.update_ruler()
 
+/mob/living/carbon/xenomorph/king/add_to_hive(datum/hive_status/HS, force=FALSE) // override to ensure proper king/hive behaviour
+	. = ..()
+	if(HS.living_xeno_king) // theres already a king
```

Comment:
You need to define a living_xeno_king as said by the check failure.

Check failure on line 261 in code/modules/mob/living/carbon/xenomorph/hive_datum.dm
GitHub Actions / Run Linters

code/modules/mob/living/carbon/xenomorph/hive_datum.dm#L261

undefined field: "living_xeno_king" on /datum/hive_status